### ISSUE TITLE
RepositoryCreateBuildInput.configOverride?

### ISSUE THREAD

**Issue by astronouth7303** [cirruslabs/cirrus-ci-web#116]
`RepositoryCreateBuildInput.configOverride` in the GraphQL schema is just listed as a "String" with no indication of how it works or what I should put in there?

**Comment by fkorotkov** [comment]
It's just a raw string of `.cirrus.yml`.

**Comment by astronouth7303** [comment]
So, wait, I can just submit an arbitrary build pipeline for a given project?

**Comment by fkorotkov** [comment]
@astronouth7303 exactly. Useful for debugging, etc. You need to have `write` permissions to the repository though.

**Comment by astronouth7303** [comment]
So only personal authentication, not organization tokens.
